Signal/src/view controllers/AdvancedSettingsTableViewController.m

```diff
@@ -52,18 +54,27 @@ - (void)loadView {
     [super loadView];
 
     self.title = NSLocalizedString(@"SETTINGS_ADVANCED_TITLE", @"");
-
+    
+    // WebRTC
+    self.enableWebRTCCell                        = [[UITableViewCell alloc] init];
```

Comment:
Yeah, that's what I'm doing and it's not changing this.  What's your objection to the formatting? I can try to fix it manually and see what clang-format thinks of it.